### DISCUSSION TITLE
请问smartDNS有跟进HTTP3的计划吗？

### DISCUSSION THREAD

**Discussion by Nautilus-Erkkinen** [pymumu/smartdns#974] [General]
http3是新一代互联网协议，底层采用quic。这类协议的延迟应该比基于tcp的dot和h2 doh更低一些，国内阿里云已经对h3提供了支持。请问有跟进计划吗？谢谢。

**Comment by pymumu** [comment]
目前没有计划，当前openssl还不支持quic，用其他实现工作量会非常大。等后续openssl支持后，再增加支持。

**Reply by LisonFan** [reply]
https://www.openssl.org/blog/blog/2023/11/23/OpenSSL32/

> OpenSSL 3.2.0 正式版发布，客户端 QUIC 支持 / 证书压缩 / 支持原始公钥等
